src/test/test_controller.c

```diff
@@ -1689,6 +1689,42 @@ test_download_status_bridge(void *arg)
   return;
 }
 
+static void
+test_getinfo_helper_dir(void *arg)
+{
+  /* We just need one of these to pass, it doesn't matter what's in it */
+  control_connection_t dummy;
+  /* Get results out */
+  char *answer = NULL;
+  const char *errmsg = NULL;
+
+  (void)arg;
+
+  setup_bridge_mocks();
+
+  getinfo_helper_dir(&dummy,
```

Comment:
Nice start! However this unittest does not actually test that the right consensus is returned. Ideally the unittest here should make sure that the right consensus string will be returned. (Hint: This might require some mocking)